docs/README.md

```diff
@@ -4,15 +4,21 @@ Use [Just the Docs](https://github.com/just-the-docs/just-the-docs) for document
 
 ## License
 
-Uses [Just the Docs](https://github.com/just-the-docs/just-the-docs)
+In order to build the documentation, we use the [Just the
+Docs](https://github.com/just-the-docs/just-the-docs) theme for Jekyll, which
+is licensed under the [MIT license](LICENSE.just-the-docs.txt).  Please see
+that file for the theme’s copyright and permission notices.
```

Comment:
Is it weird that `LICENSE.just-the-docs.txt` is under the `docs` directory instead of the `licenses` directory?

Reply:
If we move it to the `licenses` directory, the file should be renamed `LICENSE-just-the-docs.txt` to be consistent with similar files.

Reply:
I'll double-check to see whether we link to it from anywhere in the documentation (and thus, whether it needs to be where it is), and if not, I'll move it to `licenses` as you suggest.

Reply:
Moved and renamed the license, changed the link to it in the documentation `README`, and removed the old path from the Jekyll excludes.